arbnode/node.go

```diff
@@ -475,6 +478,9 @@ func ConfigAddOptions(prefix string, f *flag.FlagSet, feedInputEnable bool, feed
 
 	archiveMsg := fmt.Sprintf("retain past block state (deprecated, please use %v.caching.archive)", prefix)
 	f.Bool(prefix+".archive", ConfigDefault.Archive, archiveMsg)
+	f.Bool(prefix+".pprof", ConfigDefault.EnablePprof, "enable pprof profiling for Go")
```

Comment:
I saw the recommended semantics for enabling things is doing `--node.something.enable`, but when doing this, it seems to want another config grouping of things. Thought it was overkill to create a flag grouping just for pprof options, but could change to that if desired for consistency

Reply:
Since there's host and port options I think it would make sense to create s grouping here, also for consistency as you mentioned

Reply:
That way you can just call `startPprof(nodeConfig.Node.Pprof) and all the logic is in one place.